src/Tools/LeanCode.StyleCop/LeanCode.StyleCop.csproj

```
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="StyleCop.Analyzers" PrivateAssets="None" />
+    <PackageReference Include="StyleCop.Analyzers" PrivateAssets="None"  Version="1.1.118"/>
```

```suggestion
    <PackageReference Include="StyleCop.Analyzers" PrivateAssets="None"  Version="1.2.0-beta.435" />
```

StyleCop needs to be on the newest version to work - the stable one is 4 years old.